src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -156,6 +156,9 @@ const WorkflowIOTable = ({ which, inputsOutputs: data, config, errors, onChange,
               !readOnly ? h(DelayedAutocompleteTextArea, {
                 autosize: true,
                 'aria-label': name,
+                // without quoting both the key and value for spellcheck, it won't make it to the browser
+                // eslint-disable-next-line quote-props
+                'spellcheck': 'false',
```

Comment:
Quoting `'false'` makes sense, since booleans get handled differently, but it shouldn't be necessary to quote the key. I just verified that it works without the quotes locally, could you please double check?

Reply:
@panentheos thank you! You're totally correct. I tried `spellcheck: false` which didn't work,  then `'spellcheck': false` which also didn't work, and finally `'spellcheck': 'false'` which did work, and never went back to try `spellcheck: 'false'`. Good catch.